README.md

```diff
@@ -156,3 +156,21 @@ This will run Voyager in "server-mode" sending requests to voyager-server, which
 The server url is controlled by the `SERVER` environment variable.
 
 See [voyager-server](https://github.com/vega/voyager-server) for more information on what portions of the functionality the server handles.
+
+## Docker
+Docker image based on [node:6.17.1](https://hub.docker.com/layers/node/library/node/6.17.1/images/sha256-0bb1f1235ffb537f2543de5fa0de7e4e93f522dd9cc6da51da255cffbdd99aa3?context=explore)
```

Comment:
Would it make sense to use VSCode's container development feature instead? https://code.visualstudio.com/docs/remote/containers

Reply:
I'm not familiar with VSCode's development container feature. It certainly looks interesting.

In my case, my deployment environment requires the Docker container, though I could potentially see the same container reused for development testing & debugging.

Reply:
Okay. I'll let @kanitw make the call whether he wants a docker container in master right now. Even if not, it'll be great to have this pull request as a reference if someone wants to use docker.